src/components/FlowRunResumeModal.vue

```diff
@@ -3,10 +3,18 @@
     <p-label label="Current Flow Run State">
       <StateBadge :state="flowRun.state" />
     </p-label>
-    <div>
+
+    <div v-if="inputSchema">
+      <strong>Flow requires input.</strong> Please fill out the form below to resume.
```

Comment:
Should this say `Flow run requires input`?

Reply:
Flow run seems better, yeah!